seed/data_importer/tasks.py

```diff
@@ -151,7 +155,7 @@ def do_checks(org_id, propertystate_ids, taxlotstate_ids, import_file_id=None):
         # specify the chord as an immutable with .si
         chord(tasks, interval=15)(finish_checking.si(progress_data.key))
     else:
-        finish_checking.s(progress_data.key)
+        progress_data.finish_with_success()
```

Comment:
Could someone verify this is ok to do?

If all uploaded bsync files were bad, the front-end would spin forever on checking the progress of this because the backend would always say it was not started. I couldn't figure out why calling `finishe_checking.s(progress_data.key)` wasn't setting it to finished, so I just did it here.

Reply:
Nothing seems wrong with what you have; however, I am concerned with 1) always eager vs not-always eager 2) making sure that this is not orthogonal to https://github.com/SEED-platform/seed/pull/2055

Reply:
I just ran some tests locally:

Set `CELERY_TASK_ALWAYS_EAGER = True
CELERY_TASK_EAGER_PROPAGATES = True` and not running celery then uploaded a single buildingsync file that was invalid and it still worked as expected.

Using celery it also worked. Let me know if these tests aren't sufficient

Reply:
excellent. Thanks for verifying!